orchestration/dagster_orchestration/hca_orchestration/environments/stage_data_local_e2e.yaml

```diff
@@ -2,6 +2,10 @@ resources:
   beam_runner:
     config:
       target_class: "hca-transformation-pipeline"
+  load_tag:
+      config:
+        load_tag_prefix: "fake_load_tag_prefix"
```

Comment:
Is this just a placeholder until we finish building out all of HCA ingest?

Reply:
This is for the E2E pipeline testing, so I think it should remain a "fake" tag.  I'll give this file a clearer name.